File: 프로그래머스/0/120825. 문자 반복 출력하기/문자 반복 출력하기.py
def solution(my_string, n):
    answer = []
    arr = list(map(str,my_string))
    
    print(arr)
    for i in range(len(arr)) :
        for j in range(n) :
            answer.append(arr[i])
    
    answer = ''.join(answer)
    return answer

File: 프로그래머스/0/120825. 문자 반복 출력하기/문자 반복 출력하기.java
class Solution {
    public String solution(String my_string, int n) {
        StringBuilder answer = new StringBuilder();
        for(int i = 0; i < my_string.length(); i++){
            answer.append(String.valueOf(my_string.charAt(i)).repeat(n));
            
        }
        
        return answer.toString();
    }
}